<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="14383" started-at="2016-05-13T03:53:18Z" finished-at="2016-05-13T03:53:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="14383" started-at="2016-05-13T03:53:18Z" finished-at="2016-05-13T03:53:32Z">
      <class name="TestNGOnePack.ClassTwo">
        <test-method status="PASS" signature="setup()[pri:0, instance:TestNGOnePack.ClassTwo@ece7f4]" name="setup" is-config="true" duration-ms="14234" started-at="2016-05-13T10:53:03Z" finished-at="2016-05-13T10:53:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="load_url()[pri:0, instance:TestNGOnePack.ClassTwo@ece7f4]" name="load_url" is-config="true" duration-ms="13713" started-at="2016-05-13T10:53:18Z" finished-at="2016-05-13T10:53:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- load_url -->
        <test-method status="FAIL" signature="assertion_method_1()[pri:0, instance:TestNGOnePack.ClassTwo@ece7f4]" name="assertion_method_1" duration-ms="452" started-at="2016-05-13T10:53:31Z" finished-at="2016-05-13T10:53:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Expected and actual match in assertion_method_1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Expected and actual match in assertion_method_1
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.assertNotEquals(Assert.java:831)
	at TestNGOnePack.ClassTwo.assertion_method_1(ClassTwo.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertion_method_1 -->
        <test-method status="PASS" signature="assertion_method_2()[pri:0, instance:TestNGOnePack.ClassTwo@ece7f4]" name="assertion_method_2" duration-ms="0" started-at="2016-05-13T10:53:32Z" finished-at="2016-05-13T10:53:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- assertion_method_2 -->
        <test-method status="PASS" signature="verification_method()[pri:0, instance:TestNGOnePack.ClassTwo@ece7f4]" name="verification_method" duration-ms="141" started-at="2016-05-13T10:53:32Z" finished-at="2016-05-13T10:53:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verification_method -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestNGOnePack.ClassTwo@ece7f4]" name="tearDown" is-config="true" duration-ms="3996" started-at="2016-05-13T10:53:32Z" finished-at="2016-05-13T10:53:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestNGOnePack.ClassTwo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
